code/savegame.py

```diff
@@ -533,6 +522,8 @@ def _find_attribute(obj, options, **kwargs):
     g.pl.log.clear()
     g.pl.log.extend(new_log)
 
+    g.internal_id_version = None
```

Comment:
Could we use a `try-finally` to ensure it is always reset to None.  I would like "interesting" bugs where a person loads an oldstyle savegame that throws an exception and now that value lingers for unrelated code paths.